graphql/tests/userSettings/queries/update-user-settings.graphql

```diff
@@ -0,0 +1,5 @@
+mutation {
+  updateUserSettings(
+    userSettings: { timezone: "UTC", slackUsername: "mohamed.khelif" }
+  )
+}
```

Comment:
if the tests included tests for some of the error paths in the resolver code then needing to include `err.Error()` would have made itself known :)